clients/admin-ui/src/features/dataset/EditFieldDrawer.tsx

```diff
@@ -42,29 +42,31 @@ const EditFieldDrawer = ({ field, isOpen, onClose }: Props) => {
     values: Pick<DatasetField, "description" | "data_categories">,
   ) => {
     // merge the updated fields with the original dataset
-    if (dataset && collectionIndex !== undefined && fieldIndex !== undefined) {
-      const updatedField = { ...field, ...values };
-      const updatedDataset = getUpdatedDatasetFromField(
-        dataset,
-        updatedField,
-        collectionIndex,
-        fieldIndex,
-      );
-      updateDataset(updatedDataset);
-      onClose();
-    }
+    const updatedField = { ...field!, ...values };
```

Comment:
I am this could be simplified in the future. But here I'm reusing the existing helper functions that are unit tested.